net-misc/sslh/sslh-1.18-r1.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+MY_P="${PN}-v${PV}"
+inherit toolchain-funcs systemd
+
+DESCRIPTION="Port multiplexer - accept both HTTPS and SSH connections on the same port"
+HOMEPAGE="http://www.rutschle.net/tech/sslh.shtml"
+SRC_URI="http://www.rutschle.net/tech/${PN}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~m68k ~mips ~s390 ~sh ~x86"
+IUSE="caps tcpd systemd"
+
+RDEPEND="caps? ( sys-libs/libcap )
+	tcpd? ( sys-apps/tcp-wrappers )
+	systemd? ( sys-apps/systemd )
+	dev-libs/libconfig"
+DEPEND="${RDEPEND}
+	dev-lang/perl"
+
+RESTRICT="test"
+
+S=${WORKDIR}/${MY_P}
+
+PATCHES=(
+	"${FILESDIR}/${PN}-1.17-version-deps.patch"
```

Comment:
I've had trouble with this issue for a while (I can only ever reproduce it the first time, I guess due to caching), but your ``sslh-1.18-version-deps.patch`` seems to be a better fix so I would consider dropping the 1.17 version.